demo/restoration_video_demo.py

```diff
@@ -57,15 +62,23 @@ def main():
     model = init_model(
         args.config, args.checkpoint, device=torch.device('cuda', args.device))
 
-    output = restoration_video_inference(model, args.input_dir,
-                                         args.window_size, args.start_idx,
-                                         args.filename_tmpl, args.max_seq_len)
+    output, fps = restoration_video_inference(model, args.input_dir,
+                                              args.window_size, args.start_idx,
+                                              args.filename_tmpl,
+                                              args.max_seq_len)
 
     file_extension = os.path.splitext(args.output_dir)[1]
+    if args.fps is None:
+        if fps is None and file_extension in VIDEO_EXTENSIONS:
```

Comment:
The logic is different with 
'If the input is not a video, you must specify the frame rate of the output video.'

Reply:
"args.fps" is None means the user did not specify the frame rate.
"fps is None" means the input is not a video.
"file_extension in VIDEO_EXTENSIONS" means the output is a video.

There will be an error when the input is not a video and the user did not specify the frame rate of the output video. I guess it should be ok?

Reply:
OK, I see.